src/Header/Header.js

```diff
@@ -1,17 +1,36 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './Header.css'
 
-function Header() {
-  return(
-    <header className="header">
-      <img alt="Magic The Gathering" src="./magic_logo.png" className="magic-logo"></img>
-      <ul className="header--ul">
-        <li>My Cards</li>
-        <li>My Favorites</li>
-        <li>My Wish List</li>
-      </ul>
-    </header>
-  )
+class Header extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  setAsideView = event => {
+    if (event.target.classList.contains("my-cards")) {
+      this.props.setAsideView("myCardList");
+    } else if (event.target.classList.contains("fave-decks")) {
+      this.props.setAsideView("faveDecks");
+    } else if (event.target.classList.contains("my-wish-list")) {
+      this.props.setAsideView("wishList")
+    }
```

Comment:
I think we could change this if/ else to three different methods and then each on click would have a unique method to call. Let me know your thoughts!

Reply:
Great idea! this will make it easier to test too! I'll make that change when I write the tests.